src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1162,12 +1162,18 @@ m2mTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptr
 void
 hostMemBandwidthKernelTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptree& _ptTest)
 {
-  uint64_t shared_host_mem = 0;
+  auto shared_host_mem = xrt_core::device_query<xrt_core::query::shared_host_mem>(_dev);
+
   try {
-    shared_host_mem = xrt_core::device_query<xrt_core::query::shared_host_mem>(_dev);
+    xrt_core::device_query<xrt_core::query::enabled_host_mem>(_dev);
```

Comment:
let's add some comments of what we are checking
"// check if shell supports host mem"